edx/frontend-app-gradebook
# break out status alerts into their own component

**TL;DR -** break out status alerts into their own component

JIRA: [EDUCATOR-5664](https://openedx.atlassian.net/browse/EDUCATOR-5664)

**Developer Checklist**

**Testing Instructions**

Regression testing:
- All of the following should result in an error status alert appearing:
  - enter a negative value for min course grade and click apply
  - enter a value over 100 for min course grade and click apply
  - enter a negative value for max course grade and click apply
  - enter a value over 100 for max course grade and click apply
  - enter a value over 100 for both max course grade and min course grade and click apply
  - enter a negative value for both max course grade and min course grade and click apply
  - enter a value over 100 for max course grade and a negative value for min course grade and click apply
  - enter a negative value for max course grade and a value over 100 for min course grade and click apply
- Override a grade. After the spinner disappears, the success banner should appear. Performing other actions should not result in the banner disappearing. Clicking the X should remove the banner.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:



FYI: @edx/masters-devs-gta
